web/modules/custom/hello_world/hello_world.info.yml

```diff
@@ -0,0 +1,12 @@
+name: Hello World Module
+description: This is our first hello world module
+core_version_requirement: 10.x
+type: module
+package: Custom Modules
+dependencies:
+  - action
+  - admin_toolbar:admin_toolbar_search
+test_dependencies:
+  - book
+permissions:
+  - 'access custom module'
```

Comment:
Explain the use case of this with a screenshot

Reply:
![Screenshot from 2023-06-14 13-26-18](https://github.com/ankitdebnath003/Custom-Module/assets/122445989/2913ecfe-c9c8-4eee-9905-6831af36fc04)